content/blog/python-none/index.md

```diff
@@ -0,0 +1,58 @@
+```markdown
+---
+{
+    title: 'Python None',
+    description: 'Interpreted Languages And Their Footguns',
+    published: '2022-07-27T20:00:00.945Z',
+    edited: '2022-07-27T20:00:00.945ZZ',
+    authors: ['williamcook'],
+    tags: ['python', 'go', 'bugs'],
```

Comment:
We're actually trying to limit how many tags we have present, since before we restricted tags it was a bit chaotic 😅

```suggestion
    tags: ['python', 'go'],
```